io.github.ecotubehq.player.json

```
@@ -0,0 +1,317 @@
+                    }
+                }
+            ]
+        },
+        {
+            "name": "libdvdnav",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.videolan.org/pub/videolan/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2",
+                    "mirror-urls": [
+                        "https://videolan.mirror.ba/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2",
+                        "https://videolan.c3sl.ufpr.br/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2"
+                    ],
+                    "sha256": "c191a7475947d323ff7680cf92c0fb1be8237701885f37656c64d04e98d18d48",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "https://www.videolan.org/developers/libdvdnav.html",
+                        "version-pattern": "The latest version of <code>libdvdnav</code> is <b>([\\d\\-\\.]+)</b>\\.",
+                        "url-template": "https://download.videolan.org/pub/videolan/libdvdnav/$version/libdvdnav-$version.tar.bz2"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "libmpv",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuild-date=false",
+                "-Dlibmpv=true",
+                "-Dmanpage-build=disabled",
+                "-Dcdda=enabled",
+                "-Ddvbin=enabled",
+                "-Ddvdnav=enabled",
+                "-Dlibarchive=enabled",
+                "-Dsdl2=enabled",
+                "-Dvulkan=enabled",
+                "-Dvulkan-interop=enabled"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mpv-player/mpv.git",
+		    "tag": "v0.38.0"
+                }
+            ]
+        },
+        {
+          "name": "youtube-dl",
+          "no-autogen": true,
+          "no-make-install": true,
+          "make-args": ["youtube-dl", "PYTHON=/usr/bin/python3"],
+          "post-install": ["install youtube-dl /app/bin"],
+          "sources": [{
+            "type": "git",
+            "url": "https://github.com/rg3/youtube-dl.git",
+            "tag": "2020.05.29",
+            "commit": "228c1d685bb6d35b2c1a73e1adbc085c76da6b75"
+          }]
+        },
+        {
+            "name": "ecotube",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://github.com/ecotubehq/player",
+		"tag": "v01beta"
+              } 
+            ]
+        }
+    ],
+    "cleanup": ["/lib/pkgconfig", "/share/ffmpeg/examples", "/lib*/*/*/*.la", "/include", "/share/man", "/lib/*.la", "/bin", "/lib/cmake", "/lib/*/*.la", "/lib/*.a", "/sbin"]  
```

```suggestion
    "cleanup": ["/lib/pkgconfig", "/share/ffmpeg/examples", "*.la", "*.a", "/include", "/share/man", "/lib/cmake"]  
```

`/bin` and `/sbin` should be in the respective module, otherwise it will remove everything, even the main executable.